### ISSUE TITLE
file extension remains same as source?

### ISSUE THREAD

**Issue by Jasper2-0** [oncomouse/gulp-pandoc-writer#1]
When configuring your plugin as described, the destination filename remains the same as the source, while the destination type is specified... any thoughts on what might be the cause?

```
var gulp = require('gulp');
var pandocWriter = require('gulp-pandoc-writer');

var contentPaths = [
    'content/**/*.txt'
];

var icmlOutputDir = 'icml/';

/** Task -- start content watching **/
gulp.task('watch-icml',function(){
    return gulp.watch(contentPaths,['icml'])
});


/** Task -- convert content to ICML **/
gulp.task('icml',function(){
    return gulp.src(contentPaths)
    .pipe(pandocWriter({
        outputDir: icmlOutputDir,
        fileType: '.icml',
        args: [
            '--standalone',
            '--smart'
        ]
    }))
});
```

**Comment by Jasper2-0** [comment]
This is fixed in https://github.com/Jasper2-0/gulp-pandoc-writer

**Comment by oncomouse** [comment]
And merged in #2. Thanks for fixing it!
